NorthstarDLL/CMakeLists.txt

```
@@ -148,12 +151,12 @@ add_library(NorthstarDLL SHARED
             "audio_asm.asm"
             "dllmain.cpp"
             "dllmain.h"
-            "ns_version.h"
-)
+            "ns_version.h")
```

Accidental line change ^^

```suggestion
            "ns_version.h"
)
```